composer.json

```diff
@@ -107,7 +106,7 @@
         "doctrine/mongodb-odm-bundle": "To support MongoDB. Only versions 4.0 and later are supported.",
         "elasticsearch/elasticsearch": "To support Elasticsearch.",
         "ocramius/package-versions": "To display the API Platform's version in the debug bar.",
-        "phpdocumentor/reflection-docblock": "To support extracting metadata from PHPDoc.",
+        "phpstan/phpdoc-parser": "To support extracting metadata from PHPDoc.",
```

Comment:
would it be possible to create this as another PR so that we add the support of phpstan/phpdoc-parser to 3.1 and deprecate the old extraction?